odl/space/numpy_ntuples.py

```diff
@@ -1047,53 +1056,89 @@ def __repr__(self):
         return '{}({})'.format(class_name, inner_str)
 
     # Copy these to handle bug in ABCmeta
```

Comment:
We should make a new issue in that case, I tend to agree here (could also remove for geometries?)